symphony/lib/toolkit/class.entrymanager.php

```diff
@@ -408,13 +412,15 @@ public static function fetch($entry_id = null, $section_id = null, $limit = null
         // Handle when the sort field is an actual Field
         } elseif (self::$_fetchSortField && $field = FieldManager::fetch(self::$_fetchSortField)) {
             if ($field->isSortable()) {
-                $field->buildSortingSQL($joins, $where, $sort, self::$_fetchSortDirection, $select);
+                $field->buildSortingSQL($joins, $where, $sort, self::$_fetchSortDirection);
+                $sortSelectClause = $field->buildSortingSelectSQL($sort, self::$_fetchSortDirection);
```

Comment:
Minor, but what if was called `buildSortingSelectClause`, as the result is not actually SQL.